text/0000-pack-manifest-list-commands.md

```diff
@@ -0,0 +1,234 @@
+      "platform": {
+        "architecture": "amd64",
+        "os": "windows"
+      }
+    }
+  ]
+}
+```
+
+Finally, our manifest list is ready to be pushed to a registry, and we can use the `pack manifest push` command to do it:
+
+```bash
+$ pack manifest push cnbs/sample-package:hello-multiarch-universe
+```
+And our manifest list should be published at [dockerhub](https://hub.docker.com/r/cnbs/sample-package/tags) as `cnbs/sample-package:hello-multiarch-universe`, asuming that we have the proper credentials to push the image.
+
+<!--
+This provides a high level overview of the feature.
+
+- Define any new terminology.
+- Define the target persona: buildpack author, buildpack user, platform operator, platform implementor, and/or project contributor.
+- Explaining the feature largely in terms of examples.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to existing users and new users.
+-->
+
+# How it Works
+[how-it-works]: #how-it-works
+
+This is the technical portion of the RFC, where you explain the design in sufficient detail.
+
+The section should return to the examples given in the previous section, and explain more fully how the detailed proposal makes those examples work.
+
+# Migration
+[migration]: #migration
+
+This section should document breaks to public API and breaks in compatibility due to this RFC's proposed changes. In addition, it should document the proposed steps that one would need to take to work through these changes. Care should be give to include all applicable personas, such as platform developers, buildpack developers, buildpack users and consumers of buildpack images.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+
+We should decide to do not add this feature and users could use tools like `docker` or `podman` to create and handle their manifest list, however this is a poor user experience forcing users to use different tools to fulfill scenarios that are part of the business domain of pack.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- What other designs have been considered?
+
+We also have in mind to improve existing commands like `pack builder create` and `pack buildpack package` to support the creation of a manifest list without the need of a new command. However, this approach could not be suitable for buildpack authors who are maintaining existing buildpacks and they will need to change their current process to generate multi-arch images.
+
+- Why is this proposal the best?
+
+Because we will provide the tool to our end users to solve their problems without the need of using other tools.
+
+- What is the impact of not doing this?
+
+The impact of not doing this is that users will need to use other tools to create and handle their manifest list, which is a poor user experience.
+
```

Comment:
My $0.02 only, but I would really rather have `pack builder create` and `pack buildpack package` to support multi-arch natively. If this is a required step to get there, then ignore me. 

I have little desire to use these commands directly and would be totally fine using `docker` to manage manifest lists if I absolutely had to in the interim.

Reply:
Hi @dmikusa. We are working on these commands as part of a mentorship program for LFX and initially, we wanted to work on `pack builder create` and `pack buildpack package` but we are also working in parallel removing the support for `stacks`, redefining the toml files and there was some interception between initiatives that make us work on this direction first, without having to wait for `stack removal` but still adding value for end users